src/app/models/form.server.model.ts

```diff
@@ -883,6 +882,19 @@ const compileFormModel = (db: Mongoose): IFormModel => {
       return next(err)
     }
 
+    // Webhooks only allowed if encrypt mode
+    if (
+      this.responseMode !== FormResponseMode.Encrypt &&
+      this.webhook?.url &&
+      this.webhook.url.length > 0
```

Comment:
nit: can collapse into `this.webhook?.url?.length > 0`

Reply:
thanks! done